chart/templates/import-api/deployment.yaml

```diff
@@ -76,7 +76,7 @@ spec:
       {{- end }}
       containers:
         - name: import-api
-          image: {{ template "carto.importApi.image" . }}
+          image: {{ .Values.importApi.image.registry }}/{{ .Values.importApi.image.repository }}:{{ .Chart.AppVersion }}
```

Comment:
Just as a suggestion, using `{{ template "carto...` could help to delegates building the image name [to the helper](https://github.com/CartoDB/carto3-helm/blob/666b425897ac12812825189913e5520b634b9ef0/chart/templates/_helpers.tpl#L133), as you can see there is some login building the image name so the` deployment.yaml` stay clean.

The same in the rest of `image: {{ .Values....`